### PR TITLE
Cover pkg/rtc

### PR DESCRIPTION
**Pull request by MDr164**
Signed-off-by: Marvin Drees <marvin.drees@9elements.com>

### CODE REVIEW

File: pkg/rtc/rtc_linux_test.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+			name: "error",
+		},
+	} {
+		t.Run(tt.name, func(t *testing.T) {
+			f, err := ioutil.TempFile("", "rtc-*")
+			if err != nil {
+				t.Error(err)
+			}
+			devs = []string{f.Name()}
+			f.Close()
```
Comment (walterchris) [inline pkg/rtc/rtc_linux_test.go:34]: defer?

File: pkg/rtc/rtc_linux_test.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
```
Comment (walterchris) [inline pkg/rtc/rtc_linux_test.go:84]: defer?

File: pkg/rtc/rtc_test.go [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+		},
+	} {
+		t.Run(tt.name, func(t *testing.T) {
+			if tt.name == "open rtc" {
+				f, err := ioutil.TempFile("", "rtc-*")
+				if err != nil {
+					t.Error(err)
+				}
+				devs = []string{f.Name()}
+				f.Close()
```
Comment (walterchris) [inline pkg/rtc/rtc_test.go:36]: defer?

**Review by walterchris — COMMENTED** [review]
Not sure if it would be smarter to use defer. There is a t.Error before f.Close - does t.Error break/continue ?

**Review by walterchris — DISMISSED** [review]

**Review by depate — APPROVED** [review]
lgtm

**Review by jensdrenhaus — APPROVED** [review]

**Comment by ChriMarMe** [comment]
This may need a rebase

**Pull request closed without merging**